### ISSUE TITLE
follow-up round of interviews or survey

### ISSUE THREAD

**Issue by hannahkane** [mozilla/mattermoz#89]
Now that we've been offering Chat for awhile, should we follow up with staff to see why they're using / not using it?

**Comment by alanmoo** [comment]
@hannahkane This is a good idea- Any suggestions on how we should go about this? One red flag I saw was its lack of use at All-Hands, though I wonder if that might change now that push notifications work.

**Comment by hannahkane** [comment]
hmmm...thinking on this more, I'm more interested in the people who aren't using it. What about just identifying a handful of people who don't seem to be using it much, and having 1:1 conversations with them?

**Comment by alanmoo** [comment]
Good idea. @cadecairos is there a query you can run to see who uses Chat the least/hasn't logged in recently?
